.github/workflows/build-and-publish.yml

```diff
@@ -213,34 +153,24 @@ jobs:
         with:
           name: linux
 
-      - name: Download Windows Artifacts
-        uses: actions/download-artifact@v1
-        with:
-          name: windows
-
-      - name: Upload Windows CLI to S3
-        run: |
-          aws s3 cp ./windows/ev-enclave-${{ env.WINDOWS_TARGET }}-${{ inputs.full-version }}.tar.gz s3://cage-build-assets-${{ inputs.stage }}/cli/${{ inputs.major-version }}/${{ inputs.full-version}}/${{ env.WINDOWS_TARGET }}/ev-enclave.tar.gz
-
       - name: Upload MacOS CLI to S3
         run: |
-          aws s3 cp ./macos/ev-enclave-${{ env.MACOS_TARGET }}-${{ inputs.full-version }}.tar.gz s3://cage-build-assets-${{ inputs.stage }}/cli/${{ inputs.major-version }}/${{ inputs.full-version }}/${{ env.MACOS_TARGET }}/ev-enclave.tar.gz
+          aws s3 cp ./macos/ev-enclave-${{ env.MACOS_TARGET }}-${{ inputs.full-version }}.tar.gz s3://evervault-cli/cli/${{ inputs.stage }}/${{ inputs.major-version }}/${{ inputs.full-version }}/${{ env.MACOS_TARGET }}/ev-enclave.tar.gz
```

Comment:
Moving this into a `/cli` dir, so we don't mix things up with the existing cli stuff. 

Also adding a stage subdirectory (we didn't have a staging cli previously, except for enclaves)

Reply:
`cli.evervault.com/cli` isn't super clean though, open to suggestions